targets/lenovo-x1-carbon.nix

```diff
@@ -311,6 +313,11 @@
     {
       ghaf.host.secureboot.enable = false;
     }
+    # We can set it to true, if no Yubico hardware connected it works in usual way
+    ../modules/host/yubico.nix
+    {
+      ghaf.host.passwordless_authentication.enable = true;
```

Comment:
Can't make this default to `true` yet.

Reply:
Above might be outdated request. I noticed the line 316 comment just know but can test this later.

Reply:
We can set default to false if required.